### ISSUE TITLE
⌨ Accessibility Issue: Missing image tag on the landing page 

### ISSUE THREAD

**Issue by Elisa0122** [GCTC-NTGC/gc-digital-talent#6232]
## ❌ Current behaviour (issue)

The banner image is hidden for a blind user on the landing page.  The image does not communicate key GC DT visual information regarding the type of inclusion we practice to the screen reader user, excluding them form this message. 

**WCAG 2.1 Criteria:** 1.1.1 non-text content - Text alternatives for any non-text content in forms people need, such as alt alternative text, large print, braille, speech, symbols or simpler language.

### How to reproduce
1. Go to https://uat-talentcloud.tbs-sct.gc.ca/en
2. use apple VoiceOver on chrome
3. Start reading by using VO + A

## ✔ Expected behaviour (fix)
1. The image is read and the visual message it communicates is described.
2. The alt attribute of the random image is there: "A diverse group of people, representing all races, genders, and backgrounds, gathered together in unity. Everyone is welcome here!"

### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.
- [ ] Apple Voice Over screen reader can programmatically determine that there is an image.
- [ ] Apple Voice Over screen reader user hears "A diverse group of people, representing all races, genders, and backgrounds, gathered together in unity. Everyone is welcome here!"

## Additional information (optional)

1. How does it impact users?
The image is an important part of the content. It is important to include descriptive alt text for images to ensure that all users, regardless of their abilities or technical limitations, can access and understand the content on the web.

2. Related issues, suggestions how to fix
The image must be read and described. The alt attribute of the random image is: "A diverse group of people, representing all races, genders, and backgrounds, gathered together in unity. Everyone is welcome here!"
